CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- Option to get the age of a item in the tubes. add age to the --stats
+  See https://gist.github.com/derkgort/d05fba4368aed01c7908ee26366d7dda for change in effect
```

Comment:
Sorry I was not clear this should just go in a comment in the PR. I merged and fixed this up.